docs/source/changelog.rst

```diff
@@ -1,6 +1,33 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2019-09-15
+-----------------------------
+
+This minor release would bring improvements and new feature additions to the already existing symbolic calculations module.
+Part of this release is sponsored by European Space Agency, through Summer of Code in Space
+(SOCIS) 2019 program.
+
+Features
+........
+
+* New tensors in symbolic module
+
+  * Ricci Scalar
+  * Weyl Tensor
+  * Stress-Energy-Momentum Tensor
+  * Einstein Tensor
+  * Schouten Tensor
+
+* Improvement in performance of current tensors
+
+  * Lambdify option for tensors
+  * Support for vectors at arbitrary space-time symbolically as 1\ :sup:`st` order tensor.
+  * Support for scalars at arbitrary space-time symbolically as 0\ :sup:`th` order tensor.
+  * Addition of constants sub-module to symbolic module
```

Comment:
The above four points are not intended to be sub-points under `Improvement in performance of current tensors`.
```
* Improvement in performance of current tensors
* Lambdify option for tensors
* Support for vectors at arbitrary space-time symbolically as 1\ :sup:`st` order tensor.
* Support for scalars at arbitrary space-time symbolically as 0\ :sup:`th` order tensor.
* Addition of constants sub-module to symbolic module
```

Reply:
After this, please squash n rebase :)

Reply:
So, `Improvement in performance of current tensors` make reference only to the issue: use symmetry in Christoffel to decrease computing time?